datapackage.json

```diff
@@ -1080,7 +1080,7 @@
       "scheme": "file",
       "format": ".arrow",
       "mediatype": "application/vnd.apache.arrow.file",
-      "bytes": 662832,
+      "bytes": 1600864,
```

Comment:
> * The column data types have changed. In particular, 64-bit integers are now being used for the `delay` and `distance` fields. Previously I believe 16-bit integers were used. This change (1) bloats the file size significantly and (2) might break or complicate downstream use for some users as Arrow-JS will now return `BigInt` values rather than `number` values for those columns.

@jheer
You were quite right, this reduced the file size from `4_800_864` -> `1_600_864`